### DISCUSSION TITLE
Add DataDog to Kamal - Blog Post

### DISCUSSION THREAD

**Discussion by liamvdv** [basecamp/kamal#543] [Show and tell]
Hey Kamal Community,
running Kamal in production is lacking server, container and traffic monitoring. It would also be great to have tracing. This blog posts explains how you can setup the DataDog Agent for monitoring of your servers. 

- https://liamv.dev/posts/add-datadog-to-kamal/ 

Best,
Liam

**Comment by saiqulhaq** [comment]
since the new kamal doesn't use Traefik anymore, is it easy to setup datadog with new kamal version? @liamvdv
